services/gateway/pkg/config/config.go

```diff
@@ -76,9 +76,9 @@ type GRPCConfig struct {
 }
 
 type StorageRegistry struct {
-	Driver              string   `yaml:"driver"` // TODO: configure via env?
-	Rules               []string `yaml:"rules"`  // TODO: configure via env?
-	JSON                string   `yaml:"json"`   // TODO: configure via env?
+	Driver              string   `yaml:"driver" env:"GATEWAY_STORAGE_REGISTRY_DRIVER" desc:"The driver name of the storage registry to use"`
+	Rules               []string `yaml:"rules" env:"GATEWAY_STORAGE_REGISTRY_RULES" desc:"The rules for the storage registry"`
+	JSON                string   `yaml:"json" env:"GATEWAY_STORAGE_REGISTRY_CONFIG_JSON" desc:"Additional configuration for the storage registry in json format"`
```

Comment:
```suggestion
	Driver              string   `yaml:"driver" env:"GATEWAY_STORAGE_REGISTRY_DRIVER" desc:"The driver name of the storage registry to use."`
	Rules               []string `yaml:"rules" env:"GATEWAY_STORAGE_REGISTRY_RULES" desc:"The rules for the storage registry."`
	JSON                string   `yaml:"json" env:"GATEWAY_STORAGE_REGISTRY_CONFIG_JSON" desc:"Additional configuration for the storage registry in json format."`
```
Missing dots at the end of the sentence.
Wondering what the defaults and possible values like driver or rule names could be...

Reply:
I'm sure @dragotin will tell us once he has found out :wink: